src/main/java/org/wise/portal/presentation/web/controllers/student/StudentAPIController.java

```diff
@@ -433,4 +442,8 @@ HashMap<String, Object> canBeAddedToWorkgroup(Authentication auth,
     response.put("workgroupMembers", members);
     return response;
   }
+
+  private Boolean isUserAlreadyInAnotherWorkgroup(Workgroup workgroup1, Workgroup workgroup2) {
+    return workgroup1 != null && workgroup2 != null && workgroup1.getId() != workgroup2.getId();
+  }
```

Comment:
Should this function return boolean (primitive) since we're only interested in true/false and not the Boolean api? And is there a better name for this function? The function has nothing to do with the user. Something like ```isNotTheSame()```?